client/src/components/Dashboard/AddItemForm.js

```diff
@@ -57,16 +51,18 @@ const useStyles = makeStyles(theme => ({
 function mapListsToMenuItems(shoppingLists) {
   return shoppingLists.map(sl => (
```

Comment:
sl for single list?

Reply:
Yes, it stands for a single shopping list.

Reply:
for shopping list...i think